CONTRIBUTING.md

```diff
@@ -107,7 +108,8 @@ $ git remote add upstream https://github.com/thanos-io/thanos.git
 $ git remote update
 $ git merge upstream/master
 $ make build
-$ ./thanos -h
+$ export PATH=$PATH:$GOPATH/bin
+$ thanos -h
```

Comment:
@prmsrswt `thanos -h` could be replaced with `thanos -h flags/command/args` 
Also, additionally we can link it to `scripts/quickstart.sh` to get the basic idea of which components to run :) 

WDYT @daixiang0 @bwplotka ?

Reply:
> thanos -h could be replaced with thanos -h flags/command/args

What is ` thanos -h flags/command/args`? :thinking: 

> Also, additionally we can link it to scripts/quickstart.sh to get the basic idea of which components to run :)

Yes!

Reply:
> > thanos -h could be replaced with thanos -h flags/command/args
> 
> What is ` thanos -h flags/command/args`? 🤔
 
We need to mention the arguments/flags/commands whichever we are using after `thanos -h` 
for ex : `sidecar` , `store` are inculded under commands :)

Reply:
hmmm you can but what's wrong with printing all of them? (: `thanos -h` just works and essentially it's the way to test if you build Thanos properly for a start.

Reply:
I think the sub command test would be done with `make docs`, as @bwplotka said, it is enough for a start.